.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: ansible-test
+
+on:
+  pull_request:
+    paths:
+      - 'ansible/**'
+      - 'facts/**'
+      - 'tests/**'
+  push:   # This is only run when PRs are merged into master
+    branches:
+      - master
+  workflow_dispatch:
+
+
+jobs:
+  validate_datafiles:
+    name: validate_datafiles
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: facts/
+    container:
+      - image: kylerisse/ansible-tester@sha256:f6a9507ec1d7a2dd0570cd88cb19746840c4f97e7618f53158616b9e1f5a3618
+    steps:
+      - name: checkout
+        id: checkout
+        uses: actions/checkout@v2
+      - name: lint_python_files
+        run: pylint *.py
+      - name: datafile_unittest
+        run: pytest -vv
+  ansible_tests:
+    name: ansible_tests
+    runs-on: ubuntu-latest
+    container:
+      - image: kylerisse/ansible-tester@sha256:f6a9507ec1d7a2dd0570cd88cb19746840c4f97e7618f53158616b9e1f5a3618
```

Comment:
would love to move these images to a `socallinuxexpo` docker hub account if we can get one setup.

Reply:
What is required in order to do so?

Reply:
Functionally: just somebody doing it. Ideally: that person is @irabinovitch and we can tie a `socallinuxexpo` Github repo, rather than mine https://github.com/kylerisse/docker-ansible-tester. We've talked about this for a while, similar to having some tech team specific year round infrastructure like an S3 bucket or VPS to host large files that are currently in Rob's personal S3. Kind of all under the same umbrella item.

Reply:
The S3/VPS discussion is captured in https://github.com/socallinuxexpo/scale-network/issues/286

I could have sworn we had one for dockerhub/container artifact storage but didnt seem to be the case in the backlog so I spun out: https://github.com/socallinuxexpo/scale-network/issues/418